src/transaction/locator_sr.c

```diff
@@ -6274,12 +6285,35 @@ locator_delete_force_internal (THREAD_ENTRY * thread_p, HFID * hfid, OID * oid,
       /* remove query result cache entries which are relevant with this class */
       if (!QFILE_IS_LIST_CACHE_DISABLED)
 	{
+	  OID superclass_oid;
```

Comment:
Please apply hgryoo's comments here as well

OID superclass_oid = OID_INITIALIZER;

Reply:
OID_SET_NULL is better readability?